packages/core/src/drawer/drawer.stories.ts

```diff
@@ -0,0 +1,89 @@
+import type { Drawer } from './drawer.js';
+import type { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import '../button/register.js';
+import './register.js';
+
+export default {
+  title: 'Drawer',
+  args: {
+    attachment: 'right',
+    buttonSize: 'sm'
+  },
+  argTypes: {
+    attachment: {
+      control: 'radio',
+      options: ['right', 'left', 'top', 'bottom']
+    },
+    buttonSize: {
+      control: 'radio',
+      options: ['sm', 'md', 'lg']
+    }
+  }
+};
+
+const onClick = (event: Event & { target: HTMLElement }): void => {
+  (event.target.nextElementSibling as Drawer).showModal();
+};
+
+export const API: StoryObj = {
+  render: ({ attachment, buttonSize }) => html`
+    <sl-button @click=${onClick}>Show Drawer</sl-button>
+    <sl-drawer .attachment=${attachment} .closeButtonSize=${buttonSize}>
+      <h2 slot="title">In this sidepanel you can find a lot of info</h2>
+      <p>
+        Apple pie chocolate jelly-o carrot cake gummi bears halvah cake cheesecake. Sesame snaps macaroon shortbread
+        cheesecake muffin soufflé. Powder croissant sugar plum candy canes cupcake chupa chups cake marzipan. Chocolate
+        bar pie jujubes chocolate powder jelly. Marshmallow biscuit bear claw cookie topping. Tart sugar plum toffee
+        gingerbread macaroon danish brownie. Candy canes dragée sesame snaps lollipop ice cream.
+      </p>
+    </sl-drawer>
+  `
+};
+
+export const DisableClose: StoryObj = {
+  render: () => html`
+    <sl-button @click=${onClick}>Show Drawer</sl-button>
+    <sl-drawer disable-close>
+      <span slot="title" id="title">Drawer title</span>
```

Comment:
Can we add the `id="title"` automatically perhaps? Otherwise people may forget to set it?